example/e2e/detoxTest.spec.js

```diff
@@ -185,7 +185,7 @@ describe('Example', () => {
       await userChangesMinuteValue();
       await userTapsOkButtonAndroid();
 
-      await expect(timeText).toHaveText('22:30');
+      await expect(timeText).toHaveText('23:30');
```

Comment:
why this change?

Reply:
In e2e tests, the library uses moment.utc(date).format('HH:mm') to calculate the selected time in utc. Also, the way timezoneOffsetInMinutes should work is that the UTC time must be same in any timezone. That's why, the expected time should be '23:30' instead of '22:30'.